### PR TITLE
Update modbus markdown typo

### PR DESCRIPTION
**Pull request by ganey**
## Proposed change
Update "binary_ensor2" to "binary_sensor2" in example config

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information
N/A

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @ganey, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#18805).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by janiversen** [comment]
This is in the example section and is a perfect legal name, so why change it?

File: source/_integrations/modbus.markdown [hunk 1]
```diff
@@ -298,7 +298,7 @@ modbus:
         address: 100
         scan_interval: 20
         slave: 1
-      - name: "binary_ensor2"
+      - name: "binary_sensor2"
```
Comment (janiversen) [inline source/_integrations/modbus.markdown:301]: What is wrong with the name? it is a perfect legal name for a sensor. I would not bother changing it, we have another plan of changing all the names to not include the entity type (like mea1 etc.) so nobody thinks that is a demand.

**Review by janiversen — CHANGES_REQUESTED** [review]
I do not see a need to merge this commit.

But @frenck is our “gatekeeper” so I will leave it up to him.

File: source/_integrations/modbus.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/modbus.markdown:301]: It doesn't change anything functional, but it does look unprofessional to have typos. I'll therefore going to merge the change.

**Review by frenck — APPROVED** [review]
Thanks, @ganey 👍

**Pull request merged**